youtube_dl/extractor/dumpert.py

```diff
@@ -3,13 +3,14 @@
 
 from .common import InfoExtractor
 from ..utils import (
+    determine_ext,
     int_or_none,
     qualities,
 )
 
 
 class DumpertIE(InfoExtractor):
-    _VALID_URL = r'(?P<protocol>https?)://(?:(?:www|legacy)\.)?dumpert\.nl/(?:mediabase|embed|item)/(?P<id>[0-9]+[/_][0-9a-zA-Z]+)'
+    _VALID_URL = r'(?P<protocol>https?)://(?:(?:www|legacy)\.)?dumpert\.nl(?:/(?:mediabase|embed|item)/|(?:/toppers|/latest|/?)\?selectedId=)(?P<id>[0-9]+[/_][0-9a-zA-Z]+)'
```

Comment:
Also, support https://www.dumpert.nl/latest/?selectedId=100031688_b317a185.

```suggestion
    _VALID_URL = r'(?P<protocol>https?)://(?:(?:www|legacy)\.)?dumpert\.nl(?:/(?:mediabase|embed|item)/|(?:/toppers|/latest)?/?)\?selectedId=)(?P<id>[0-9]+[/_][0-9a-zA-Z]+)'
```
Are there any other "slugs" like `toppers`? We seem to get the same video (based on the `id`) regardless, but random choices of https://www.dumpert.nl/nieuw?selectedId=100031688_b317a185, etc, don't.